tests/examples/opinions/united_states/bia_extract_from_text_2.txt

```diff
@@ -0,0 +1,41 @@
+Cite as 27 I&N Dec. 48 (BIA 2017) Interim Decision #3894
```

Comment:
These fixtures are big. Can you winnow them down to just the relevant parts?

Reply:
yes

Reply:
Im always somewhat concerned that a fixture or regex pattern might be thrown off by a large PDF document, so I always make them larger as proof but I understand that they are large.  If we are shrinking them then I think it might be smarter to just figure a different way to do this.

Reply:
Not a strong feeling here. I appreciate small ones because they focus my attention on what they're trying to test, rather than the gazillion other things (and they keep repos small, etc). But really, just a slight preference here. Maybe I shouldn't care.

Reply:
√